_posts/2021-08-18-debezium-ui-availability.adoc

```diff
@@ -0,0 +1,107 @@
+
+==== Create Connector - Connector type (required)
+Choose the type of connector in step 1.  Currently the *MongoDB, MySQL and PostgreSQL* connector types are supported.  Addition of more connector types is currently in progress.
+
+[.centered-image.responsive-image]
+====
+++++
+<img src="/assets/images/2021-08-18-debezium-ui/CreateConnectorStep1.png" class="responsive-image">
+++++
+====
+
+==== Create Connector - Properties (required)
+The basic connection properties for the selected connector are entered in step 2, and the properties must be validated before proceeding.  Advanced connection properties are also provided in a separate section of this step.  Upon successful validation, the user may proceed to the next steps (Additional properties) - or they can elect to bypass the additional properties and proceed directly to Review.
+
+[.centered-image.responsive-image]
+====
+++++
+<img src="/assets/images/2021-08-18-debezium-ui/CreateConnectorStep2.png" class="responsive-image">
+++++
+====
+
+==== Create Connector - Additional properties (optional)
+The Additional properties are optional and can be summarized as follows:
+
+* *Filter definition* - entry of *regular expressions* which define the filters for inclusion/exclusion of the items that will be included for CDC.  The included items are displayed as the filters are entered and applied.
+* *Data options* - *Snapshot* and *Mapping* properties (optional).  The defaults can be viewed and changed if desired.
+* *Runtime options* - *Engine* and *Heartbeat* properties (optional).  The defaults can be viewed and changed if desired.
+
+==== Create Connector - Review
+The *Review* step provides a summary of the configuration that will be used to create the connector.  If happy with the selections, click 'Finish' to create the connector.  If the properties need adjustment, navigate back to the earlier steps.
+
+== Design Considerations
+
+The UI is implemented as a https://quarkus.io/[Quarkus]-based web application.  The backend is configured with the URL(s) of one or more Kafka Connect clusters and provides a REST interface for the frontend.  The frontend user interface uses https://reactjs.org/[ReactJS] as the primary technology, utilizing https://www.patternfly.org/v4/[Patternfly] react components and design patterns.
+
+As with everything in Debezium, the UI is fully open source (Apache License Version 2.0).  You can find the https://github.com/debezium/debezium-ui/[UI Source Code] under the Debezium organization on Github.
+
+== Trying It Out Yourself
+
+==== debezium-ui docker image
+
+The debezium-ui https://hub.docker.com/r/debezium/debezium-ui[docker image] is available for running the UI.  To start the UI and connect to an existing Kafka Connect instance:
+
+----
+$ docker run -it --rm --name debezium-ui -p 8080:8080 -e KAFKA_CONNECT_URI=http://connect:8083 debezium/debezium-ui:1.5
```

Comment:
Some users had issues with `ui.base.uri` property / `UI_BASE_URI` which is per default set to `http://localhost:8080/api`. @indraraj @uidoyen  I know one of you worked on that. Is this still needed and do we need to add some documentation to the README and the blog post on that?

Reply:
not sure, are you talking about this one [DBZ-3209](https://issues.redhat.com/browse/DBZ-3209)  ?

Reply:
Yes, maybe. But maybe I am wrong that this related?

Reply:
yeap some users reported issues around it and I am also not clear if this fixes it or it's even related. But maybe we can add some documentation to the README like how `UI_BASE_URI` by default set to `http://localhost:8080/api` and where if it can be modified.